src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import Big from 'big.js';
+
+const Operate = (num1, num2, operation) => {
+  const n1 = Big(num1);
+  const n2 = Big(num2);
+  if (operation === '-') {
+    return n1 - n2;
+  } if (operation === '+') {
+    return n1 + n2;
+  } if (operation === '/' || operation === '÷') {
+    return n1 / n2;
+  } if (operation === '*') {
+    return n1 * n2;
+  } if (operation === '%') {
+    return n1 % n2;
+  }
```

Comment:
- [ ] Also note that this-%- is not a modulus operator but a percentage operator. Please make the change and divide by 100 or multiply by 0.01.